index.js

```diff
@@ -20,7 +20,10 @@ var filesLoadedBeforeTrace = Object.keys(require.cache);
 
 // Load continuation-local-storage first to ensure the core async APIs get
 // patched before any user-land modules get loaded.
-require('continuation-local-storage');
+if (require('semver').satisfies(process.version, '<8') &&
```

Comment:
In the future if `semver` adds a dependency, it would not be caught by the `filesLoadedBeforeTrace` and the requisite test.